src/frontend/src/setupTests.js

```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import "@testing-library/jest-dom";
```

Comment:
Since you removed the testing dependencies, including `@testing-library/jest-dom`, I think you should just remove this file completely. Having tests eventually is a good idea, but for now I think this file just makes things more confusing.